vsintegration/src/FSharp.Editor/Common/Vs.fs

```diff
@@ -91,9 +92,36 @@ module internal VsRunningDocumentTable =
 
 [<AutoOpen>]
 module internal ServiceProviderExtensions =
-    type internal System.IServiceProvider with
```

Comment:
So once again, what's the need for this change? I'm not against it, just a bit paranoid and trying to better understand all this magic